ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/e2e/casework-regression/CaseActions/srma.cy.ts

```diff
@@ -596,4 +596,31 @@ describe("Testing the SRMA case action", () => {
 			.withYearReportSentToTrust("2021")
 			.save();
 	}
+
+	it("Should NOT show hint text", () => {
```

Comment:
Can we put the check for hint text in the existing test, otherwise it will slow down the suite